Added pgfog.com to the public suffix list

dmyers opened the pull request:
[Pagefog](http://pagefog.com) is a hosting service that provisions cloud servers and sets up website hosting, DNS with the `pgfg.com` domain, and SSL certificates with LetsEncrypt.

## Verification

```bash
$ host -t TXT _psl.pgfog.com
_psl.pgfog.com descriptive text "https://github.com/publicsuffix/list/pull/141"
```

sleevi commented:
Can you please explain the purpose of this addition, as explained in [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines)

Also, can you please update the TXT record to be `_psl` (note the leading underscore)

dmyers commented:
@sleevi Sorry I had looked at an old PR and used it as my reference. I've updated the original post with purpose and updated TXT record. Thank you for your help.

sleevi commented:
Please see above about explaining a bit more about the purpose of this PR and how it's used, so that we can help ensure it's properly formed.

sleevi commented:
Oh, you edited the original. Got it. Github did not email those comments.

sleevi commented:
@dmyers This request still needs to be properly sorted, as per the Guidelines (sorted by Company name - P < S)

dmyers commented:
@sleevi Sorry again about that I guess that slipped by me too and I assumed it was newest at the bottom. All corrected now. Thank you again for your help. 👍
